website/content/docs/concepts/workers.mdx

```diff
@@ -0,0 +1,94 @@
+desired to reduce latency or comply with security standards. Workers can be assigned [tags](/boundary/tutorials/worker-management/target-aware-workers) that Boundary 
+can [filter](/boundary/docs/concepts/filtering/worker-tags) through, to find the appropriate worker to use for a session. For example, Boundary could filter to workers 
+with tag “A”, to connect to targets in “Network A”.
+
+![IAM Example Structure](/img/worker-tags.png)
+
+## Multi-hop sessions <sup>HCP/ENT</sup>
+Most organizations want to provide access to infrastructure without exposing private networks. Many organizations also have complex network topologies requiring 
+inbound traffic to route through multiple network enclaves in order to reach the target system. 
+[Multi-hop](/boundary/docs/configuration/worker#multi-hop-worker-capabilities-hcp-ent) sessions allow two or more workers to be chained together 
+across multiple networks to form reverse proxy connections between the user and the target, even in complex networks with strict outbound-only policies.
+
+In multi-hop scenarios, there are typically 3 types of workers:
+1. **Ingress worker** - An ingress worker is a worker that is accessible by the client. The client will initiate the connection to the ingress worker.
+2. **Intermediary worker** - An (optional) intermediary worker sits between ingress and egress workers as part of a multi-hop chain.
+3. **Egress worker** - An egress worker is a worker that can access the target. The egress worker initiates reverse proxy connections to intermediary or ingress workers.
+
+<Tip>
+“Ingress”, “intermediary”, and “egress” are general ways to describe how the respective worker interfaces with resources, and a worker can be more than 1 of those 
+at a time. For example in the diagram below, the intermediary worker is also an egress worker since it can access a target. 
+</Tip>
+
+![IAM Example Structure](/img/concepts-multihop.png)
+
+Once the persistent connection chain is established between the workers, when the user requests to connect to a target host, they are automatically proxied from:
+1. Boundary client to ingress worker
+2. Ingress worker to intermediary worker (where applicable)
+3. Ingress worker to egress worker
+4. Egress worker to desired target
+
+## Deployment
+Workers are services that can run on a container or virtual machine, and should be deployed strategically within networks to provide access to targets. In 
+all editions of Boundary, workers are fully self-managed and can be deployed anywhere. In HCP Boundary, 3 HCP-managed workers are automatically deployed with the cluster. 
+
+To learn more about workers and deployment, see:
+* [Worker configuration](/boundary/docs/configuration/worker)
+* [Recommended architecture](/boundary/docs/install-boundary/recommended-architecture)
+* [Worker system requirements](/boundary/docs/install-boundary/system-requirements)
+* [Worker management tutorials](/boundary/tutorials/worker-management)
+
+
+
+
+
+
+
+
+
+
```

Comment:
@Dan-Heath There are some extra spaces here.

This "duplicate" PR does not have these spaces. Not sure what's going on here.

Reply:
Oh wow, good catch @moduli thank you. Yeah, this is some weird behavior. It said the backport failed for 0.13, so I re-ran all failed jobs and it created a second PR for the stable website. This backport to 0.13 doesn't look right either:

[3510
](https://github.com/hashicorp/boundary/pull/3510)

Reply:
This PR does not have the latest changes. The one I closed below is correct:

https://github.com/hashicorp/boundary/pull/3508

Reply:
Oh got it, that makes sense, thank you!  Looks like I can reopen #3508, I am going to try to merge that one if the checks pass since it's already correct, and close this one. Then I can make manual corrections to #3510 for the 0.13 branch.

Reply:
Sounds good. Just be aware that #3510 contains a lot of other changes outside of the doc changes. You may want to create a new branch and cherry-pick some commits.

Reply:
Yep, that makes sense.. not sure how all that got in there. Thank you @moduli !!